core/swift3Action/Dockerfile

```diff
@@ -4,16 +4,24 @@ FROM buildpack-deps:trusty
 
 ENV DEBIAN_FRONTEND noninteractive
 
-# Upgrade and install Python dependencies
+# Upgrade and install Python and helpers
 RUN apt-get -y purge && \
     apt-get -y update && \
-    apt-get -y install --fix-missing python2.7 python-distribute python-pip
+    apt-get -y install --fix-missing python2.7 && \
+    apt-get -y install --fix-missing python2.7-dev && \
+    apt-get -y install --fix-missing python-distribute && \
+    apt-get -y install --fix-missing python-pip && \
+    apt-get -y install --fix-missing python-gevent && \
+    apt-get clean && \
+    pip install -U flask
 
```

Comment:
Is there a reason to change these? The layers were designed to maximize caching across python, swift, swift3 and black box dockerfiles.